src/scancode/cli.py

```diff
@@ -239,6 +241,24 @@ class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
+    options = []
+
+    def __init__(self, name, context_settings=None, callback=None,
+                 params=None, help=None, epilog=None, short_help=None,
+                 options_metavar='[OPTIONS]', add_help_option=True):
+        super(ScanCommand, self).__init__(name, context_settings, callback,
+                 params, help, epilog, short_help, options_metavar, add_help_option)
+        for name, callback in plugincode.post_scan.get_post_scan_plugins().items():
+            option = ScanOption(('--' + name,), is_flag=True, help=callback.__doc__.strip(), group=POST_SCAN)
+            self.options.append(option)
+
+    def get_params(self, ctx):
+        """
+        Overridden from click.Command to return plugged in options in addtion to the
```

Comment:
Yes, it returns the plugged options as well, the `self.options` attribute stores the options created in `__init__` which this method returns on every call